src/components/CommunicationsLink.js

```diff
@@ -52,7 +53,13 @@ const CommunicationsLink = props => (
                         >
                             {props.children}
                         </Pressable>
-                    ) : props.children}
+                    ) : (
+                        <Tooltip text={props.value}>
```

Comment:
Ok. It works but it takes away flexibility that comes with `props.children`. This tooltip should be passed with children.

Reply:
Okay got it. I'll add a separate prop for Tooltip.

Reply:
I think better would be to pass the tooltip with children